script.skinshortcuts/default.py

```diff
@@ -0,0 +1,397 @@
+# coding=utf-8
+import os, sys
+import xbmc, xbmcaddon, xbmcgui, xbmcplugin, xbmcvfs
+import xml.etree.ElementTree as xmltree
+import pstats
+import random
+import time
+import calendar
+from time import gmtime, strftime
+from datetime import datetime
+from traceback import print_exc
+import json as simplejson
+import urllib.request, urllib.parse, urllib.error
+import pickle
+import _thread as thread
+
+ADDON        = xbmcaddon.Addon()
+ADDONID      = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+LANGUAGE     = ADDON.getLocalizedString
+CWD          = ADDON.getAddonInfo('path')
+ADDONNAME    = ADDON.getAddonInfo('name')
+RESOURCE     = xbmc.translatePath(os.path.join(CWD, 'resources', 'lib'))
+DATAPATH     = os.path.join(xbmc.translatePath("special://profile/"), "addon_data", ADDONID)
+MASTERPATH   = os.path.join(xbmc.translatePath("special://masterprofile/"), "addon_data", ADDONID)
+
+sys.path.append(RESOURCE)
```

Comment:
please avoid appending to sys.path, kodi interpreter already add your depends to the path of this addon.
Use absolute imports:

`from resources.lib import x,y,z`